File: Model/Java/PacienteExame.java
public class PacienteExame {
    
    private String nome;
    private String dataNascimento;
    private String dataExame;
    private String periodo;
    private String telefone;
    private String email;
    private String nomeMae;
    private String tomaMedicamento;
    private String medicamento;

    public PacienteExame(){}

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getDataNascimento() {
        return dataNascimento;
    }

    public void setDataNascimento(String dataNascimento) {
        this.dataNascimento = dataNascimento;
    }

    public String getDataExame() {
        return dataExame;
    }

    public void setDataExame(String dataExame) {
        this.dataExame = dataExame;
    }

    public String getPeriodo() {
        return periodo;
    }

    public void setPeriodo(String periodo) {
        this.periodo = periodo;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getNomeMae() {
        return nomeMae;
    }

    public void setNomeMae(String nomeMae) {
        this.nomeMae = nomeMae;
    }

    public String getTomaMedicamento() {
        return tomaMedicamento;
    }

    public void setTomaMedicamento(String tomaMedicamento) {
        this.tomaMedicamento = tomaMedicamento;
    }

    public String getMedicamento() {
        return medicamento;
    }

    public void setMedicamento(String medicamento) {
        this.medicamento = medicamento;
    }

    @Override
    public String toString() {
        return "Nome: " + nome + ", Data de Nascimento=" + dataNascimento + ", Data do Exame: " + dataExame
                + ", periodo: " + periodo + ", telefone: " + telefone + ", email: " + email + ", nomeMae: " + nomeMae
                + ", Toma Medicamento: " + tomaMedicamento + ", Medicamento: " + medicamento + "\n";
    }

    
}

File: Model/Php/PacienteExame.php
<?php 
    class PacienteExame{
        private $nome;
        private $dataNascimento;
        private $dataExame;
        private $periodo;
        private $telefone;
        private $email;
        private $nomeMae;
        private $tomaMedicamento;
        private $medicamento;

        public function __construct(){}
        
        public function setNome($nome){
            $this->nome = $nome;
        }    
        public function getNome(){
            return $this->nome;
        }

        public function setDataNascimento($dataNasc){
            $this->dataNascimento = $dataNasc;
        }
        public function getDataNascimento(){
            return $this->dataNascimento;
        }

        public function setDataExame($dataExame){
            $this->dataExame = $dataExame;
        }
        public function getDataExame(){
            return $this->dataExame;
        }

        public function setPeriodo($periodo){
            $this->periodo = $periodo;
        }
        public function getPeriodo(){
            return $this->periodo;
        }

        public function setTelefone($telefone){
            $this->telefone = $telefone;
        }
        public function getTelefone(){
            return $this->telefone;
        }

        public function setEmail($email){
            $this->email = $email;
        }
        public function getEmail(){
            return $this->email;
        }

        public function setNomeMae($nMae){
            $this->nomeMae = $nMae;
        }
        public function getNomeMae(){
           return $this->nomeMae;
        }

        public function setTomaMedicamento($tomaMed){
            $this->tomaMedicamento = $tomaMed;
        }
        public function getTomaMedicamento(){
            return $this->tomaMedicamento;
        }

        public function setMedicamento($medicamento){
            $this->medicamento = $medicamento;
        }
        public function getMedicamento(){
            return $this->medicamento;
        }

        public function __toString(){
            return "Nome: {$this->nome} Data de nascimento: {$this->dataNascimento}  Data do Exame: {$this->dataExame} Periodo: {$this->periodo} 
                    Telefone: {$this->telefone} Email: {$this->email} Nome da mãe: {$this->nomeMae} Toma Medicamento {$this->tomaMedicamento} 
                    Nome do Medicamento {$this->medicamento}";
            
        }
    }

?>